cycle_en_terre_website_design/static/src/less/cycle_en_terre_website_design.less

```diff
@@ -0,0 +1,141 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+@import url('https://fonts.googleapis.com/css?family=Old+Standard+TT');
+
```

Comment:
I think that defining vars for the main color set is a good idea. :)
```
@color-orange: #f0b61c;
@color-dark-orange: #a67c0a;
@color-dark-gray: #626262;
…
…
    background-color: @color-orange;
```

Reply:
Done